mkdocs.yml

```diff
@@ -96,16 +113,17 @@ nav:
       - R:
           - External resources: training_resources/R/README.md
           - Git with RStudio: training_resources/R/git_with_RStudio.md
-      - Refactoring Golden Rules: training_resources/refactoring-guide.md
-  - Support: support.md
-  - Glossary: glossary.md
-  - Useful links: useful_links.md
```

Comment:
ah do we think this cluttered it too much?

Reply:
yeap exactly. I was thinking cut it down to the four key sections (plus Home and Site Info aka About, which are staples on all websites), which each get their own section on the long home page also. 

As I noted somewhere else, I was also thinking to move the tags off the main nav. My thinking was to aim the structure and home page primarily with new visitors in mind, asthe main comments we were getting was that there was too much stuff and people got a bit overwhelmed. And so for new visitors, the Tags page isn't going to be useful anyway as they don't know what they are looking for yet. 

Tags would be more for returning visitors or RAP-savvy folk who are looking for something specific, who would be more motivated to stay and look around anyway.